src/authManager.tsx

```diff
@@ -120,20 +120,33 @@ export const setupUserSession = async (userManager: UserManager) => {
     localStorage.removeItem('nexus__token');
   });
 
+  let user;
   try {
-    let user;
     // do we already have a user?
     user = await userManager.getUser();
-    if (user) {
-      // we've loaded a user, refresh it
+  } catch (e) {}
+
+  if (user) {
+    // we've loaded a user, refresh it
+    try {
       user = await userManager.signinSilent();
+    } catch (error) {
+      console.error(
+        '@@error',
+        "we've loaded a user, refresh it raised an error",
+        error
+      );
```

Comment:
Should we print anything to the console? Specifically, I'm wondering if it's necessary or beneficial to print things to the console in general. Does it provide any assistance or value to us or the user? Alternatively, why not use Sentry, e.g.?

Reply:
absolutely right, change it 👍